src/gengowatcher/notifier.py

```
@@ -2,91 +2,116 @@
     """
-    command = ['notify-send', title, message]
-    
+    command = ["notify-send", title, message]
+
     if icon_path and Path(icon_path).is_file():
-        command.extend(['--icon', icon_path])
-    
+        command.extend(["--icon", icon_path])
+
     try:
         subprocess.run(command, check=True)
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)